### PR TITLE
OA Tx Test

### PR DESCRIPTION
**Pull request by Joeysantoro**
Adds new OA timelock deployment and makes a test for tx:

https://etherscan.io/tx/0x67c06a7f3d7849af89a894a63baecfbe19afb31904a04fc9ffb53b926bd85e02

### CODE REVIEW

File: proposals/dao/optimisticTimelock.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { ethers } from "hardhat";
+import hre, { ethers } from "hardhat";
 import { expect } from "chai";
+const { time } = require('../../test/helpers');
```
Comment (ditchfieldcaleb) [inline proposals/dao/optimisticTimelock.ts:3]: should use import instead of require

File: proposals/dao/optimisticTimelock.ts [hunk 2]
```diff
@@ -27,20 +28,58 @@ async function run(addresses, oldContracts, contracts, logging = false) {
...
+    await tribalChiefOptimisticTimelock.queueTransaction(
+        '0x956F47F50A910163D8BF957Cf5846D573E7f87CA',
+        '0',
+        'transfer(address,uint256)',
+        '0x000000000000000000000000bc9c084a12678ef5b516561df902fdc426d9548300000000000000000000000000000000000000000000d3c21bcecceda1000000',
+        '1632873600',
+        {from: tribalChiefOptimisticMultisigAddress}
+    );
+
+    await time.increase('1000000');
```
Comment (ditchfieldcaleb) [inline proposals/dao/optimisticTimelock.ts:55]: are you sure this works? in my dive into the unit tests, I had to convert to a number before passing into time.increase

File: proposals/dao/optimisticTimelock.ts [hunk 3]
```diff
@@ -27,20 +28,58 @@ async function run(addresses, oldContracts, contracts, logging = false) {
...
+        tribalChiefOptimisticMultisigAddress
+    } = addresses;
+    
+    await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [tribalChiefOptimisticMultisigAddress],
+    });
+
+    await tribalChiefOptimisticTimelock.queueTransaction(
+        '0x956F47F50A910163D8BF957Cf5846D573E7f87CA',
```
Comment (ditchfieldcaleb) [inline proposals/dao/optimisticTimelock.ts:47]: nit, but for future should add a comment explaining what this address was (I looked it up on etherscan ofc)

**Review by ditchfieldcaleb — APPROVED** [review]

File: proposals/dao/optimisticTimelock.ts (hunk 2 above)
Comment (Joeysantoro) [reply proposals/dao/optimisticTimelock.ts:55]: Yeah the execution works

File: proposals/dao/optimisticTimelock.ts (hunk 3 above)
Comment (Joeysantoro) [reply proposals/dao/optimisticTimelock.ts:47]: ah good call, I didn't use the feiAddress var because I wanted to use the exact calldata used by OA

**Pull request merged**